### ISSUE TITLE
关于optimizer和frozen_stage的设置

### ISSUE THREAD

**Issue by pingyang97** [weiyithu/SurroundOcc#91]
您好，关于optimizer的设置，您这边设置了 'img_backbone': dict(lr_mult=0.1)，请问这个有什么特殊的说法吗？

optimizer = dict(
    type='AdamW',
    lr=2e-4,
    paramwise_cfg=dict(
        custom_keys={
            'img_backbone': dict(lr_mult=0.1),
        }),
    weight_decay=0.01)

在img_backbone这边，为什么选择frozen_stages=1呢？
    img_backbone=dict(
       type='ResNet',
       depth=101,
       num_stages=4,
       out_indices=(1,2,3),
       frozen_stages=1,
       norm_cfg=dict(type='BN2d', requires_grad=False),
       norm_eval=True,
       style='caffe',
       #with_cp=True, # using checkpoint to save GPU memory
       dcn=dict(type='DCNv2', deform_groups=1, fallback_on_stride=False), # original DCNv2 will print log when perform load_state_dict
       stage_with_dcn=(False, False, True, True)),

非常感谢您的阅读，期待能够尽快得到您会回复～

**Comment by weiyithu** [comment]
同学你好，这块是直接参考bevformer里的超参的，没有改过这个参数

**Comment by pingyang97** [comment]
> 同学你好，这块是直接参考bevformer里的超参的，没有改过这个参数

收到，感谢回复～

**Comment by zyy612** [comment]
hi, The download link for the pretrained model is broken，the page crushed and showing "Link does not exist.",，Could you share the dataset with a resolution of 200×200×16 for the generated dense occupancy labels?
